tapir/shifts/templates/shifts/shift_detail.html

```diff
@@ -185,6 +187,17 @@ <h5 class="d-flex justify-content-between align-items-center">
                                     {% endif %}
                                 </span></td>
                             {% endwith %}
+                            <!-- Requirements -->
+                            {% for key, value in slot.get_required_capabilities_dict.items %}
+                                {% if key in request.user.shift_user_data.capabilities %}
+                                    <td><span class="material-icons">check</span>{{ value }}</td>
```

Comment:
Can you make the text and the icon align a bit nicer?
Something like 
```django
<td><span class="d-flex align-items-center"><span class="material-icons">check</span><span>{{ value }}</span></span></td>
```

Reply:
Thanks, lack of HMTL knowledge